QEfficient/customop/ctx_scatter_gather_cb.py

```diff
@@ -97,11 +97,12 @@ def symbolic(
 
 @onnxscript.script(onnxscript.values.Opset("com.qualcomm.cloud", 1))
 def CtxGatherCB(
-    data: onnxscript.FLOAT, batch_index: onnxscript.INT32, ctx_indices: onnxscript.INT32
+    data: onnxscript.FLOAT, batch_index: onnxscript.INT32, ctx_indices: onnxscript.INT32, comp_ctx_len: onnxscript.INT64
 ) -> onnxscript.FLOAT:
     batch_size = ops.Gather(ops.Shape(batch_index), [0])
     num_heads = ops.Gather(ops.Shape(data), [1])
-    ctx_len = ops.Gather(ops.Shape(data), [2])
+    # ctx_len = ops.Gather(ops.Shape(data), [2])
+    ctx_len = ops.Reshape(comp_ctx_len, [1])
```

Comment:
Remove commented code, provide the brief explanation on why to do reshape here, maybe add comments on how this will help CCL case.